corehq/motech/dhis2/entities_helpers.py

```diff
@@ -198,7 +198,7 @@ def update_tracked_entity_instance(
     attempt=1
 ):
     case_updates = {}
-    for attr_id, value_source_config in case_config.attributes.items():
+    for attr_id, value_source_config in case_config['attributes'].items():
```

Comment:
no idea whether this PR was the cause, but I just saw an alert for this bug:
https://sentry.io/organizations/dimagi/issues/3825537564/?project=136860&referrer=slack
And then saw that this PR recently changed the surrounding code, so I figured I'd flag in case that warrants investigation.

Reply:
Thanks for flagging it @esoergel. I will look into it.